makefile

```diff
@@ -0,0 +1,14 @@
+test:
+    julia --color=yes pull_julia_code.jl
+    julia --color=yes runtests.jl
+compile:
+    julia --color=yes pull_julia_code.jl
+    lualatex book
+    pythontex3 book
```

Comment:
Is it really pythontex3? Does this make a difference over `pythontex`?

Reply:
🤔 	It's likely to be my local issue, since I have two texlive, two pythons installed in my system. I must confused myself at some place.